ReleaseTooling/Sources/FirebaseReleaser/Push.swift

```diff
@@ -48,12 +48,11 @@ enum Push {
         switch destination {
         case .staging:
           return "pod repo push --skip-tests --use-json \(warningsOK) \(stagingLocation) " +
-            pod.skipImportValidation() + " \(pod.podspecName()) " +
```

Comment:
Since skipping import validation is no longer pod specific, I removed the helper and just encoded the flag in the top level push commands.

Reply:
SGTM, thanks!